interface/modules/zend_modules/module/Carecoordination/src/Carecoordination/Model/CarecoordinationTable.php

```diff
@@ -174,7 +176,7 @@ public function importNewPatient($document): void
      * @param   $document_id    Document id
      */
 
-    public function importCore($xml_content)
+    public function importCore($xml_content, $doc_id = null)
```

Comment:
would just make sure this function also works if $doc_id is null (ie. skip places that need it) since there is an option to import ccda's without saving the actual ccda in the documents:
https://github.com/openemr/openemr/blob/master/contrib/util/ccda_import/import_ccda.php#L18-L22